samples/nrf9160/modem_shell/Kconfig

```diff
@@ -70,6 +70,13 @@ config MOSH_FOTA
 	help
 	  FOTA for performing software updates over-the-air for both modem and application side.
 
+config MOSH_SIMPLE_REST_CLIENT
```

Comment:
I would probably rename this based on directory and file names, i.e., MOSH_SREST_CLIENT. Just like most of our directories.

Reply:
Done.